common/common.vcxproj

```diff
@@ -83,10 +88,16 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="EventSource.inl" />
+    <MASM Include="FastJmp.asm">
+      <FileType>Document</FileType>
+      <PreprocessorDefinitions Condition="'$(Platform)'=='Win32'">_M_X86;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(Platform)'=='x64'">_M_X86;_M_X86_64;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Is it possible to define this more globally or re-use our C++ defs?  In case we want to add more in the future it would be useful to not have to put it on each asm file separately

Reply:
Unfortunately no, MASM seems to use a separate set of defines from C++. I renamed them to _M_X86_32/64 for consistency at least.